inventory/environ.py

```diff
@@ -126,150 +132,114 @@ def getSplunkApps(inv):
-    vars_scope["splunk"]["s2s_enable"] = os.environ.get('SPLUNK_S2S_ENABLE') or getValueFromDEFAULTS("splunk.s2s_enable", casting=int) or 0
-    vars_scope["splunk"]["http_port"] = getValueFromDEFAULTS("splunk.http_port", casting=int) or 8000
-    vars_scope["splunk"]["http_enableSSL"] = getValueFromDEFAULTS("splunk.http_enableSSL", casting=int) or 0
-    vars_scope["splunk"]["http_enableSSL_cert"] = getValueFromDEFAULTS("splunk.http_enableSSL_cert")
-    vars_scope["splunk"]["http_enableSSL_privKey"] = getValueFromDEFAULTS("splunk.http_enableSSL_privKey")
-    vars_scope["splunk"]["http_enableSSL_privKey_password"] = getValueFromDEFAULTS("splunk.http_enableSSL_privKey_password")
-    vars_scope["splunk"]["hec_port"] = getValueFromDEFAULTS("splunk.hec_port", casting=int) or 8088
-    vars_scope["splunk"]["hec_disabled"] = getValueFromDEFAULTS("splunk.hec_disabled", casting=int) or 0
-    vars_scope["splunk"]["hec_enableSSL"] = getValueFromDEFAULTS("splunk.hec_enableSSL", casting=int) or 1
-    vars_scope["splunk"]["hec_token"] = os.environ.get('SPLUNK_HEC_TOKEN') or getValueFromDEFAULTS("splunk.hec_token")
-    vars_scope["splunk"]["shc"] = {}
-    vars_scope["splunk"]["shc"]["enable"] = getValueFromDEFAULTS("splunk.shc.enable") or False
-    vars_scope["splunk"]["shc"]["secret"] = os.environ.get('SPLUNK_SHC_SECRET') or getValueFromDEFAULTS("splunk.shc.secret")
-    vars_scope["splunk"]["shc"]["label"] = getValueFromDEFAULTS("splunk.shc.label") or "shc_label"
-    vars_scope["splunk"]["shc"]["replication_factor"] = getValueFromDEFAULTS("splunk.shc.replication_factor", casting=int) or 3
-    vars_scope["splunk"]["shc"]["replication_port"] = getValueFromDEFAULTS("splunk.shc.replication_port", casting=int) or 4001
-    vars_scope["splunk"]["idxc"] = {}
-    vars_scope["splunk"]["idxc"]["enable"] = getValueFromDEFAULTS("splunk.idxc.enable") or False
-    vars_scope["splunk"]["idxc"]["secret"] = os.environ.get('SPLUNK_IDXC_SECRET') or getValueFromDEFAULTS("splunk.idxc.secret")
-    vars_scope["splunk"]["idxc"]["label"] = getValueFromDEFAULTS("splunk.idxc.label") or "idxc_label"
-    vars_scope["splunk"]["idxc"]["search_factor"] = getValueFromDEFAULTS("splunk.idxc.search_factor", casting=int) or 3
-    vars_scope["splunk"]["idxc"]["replication_factor"] = getValueFromDEFAULTS("splunk.idxc.replication_factor", casting=int) or 3
-    vars_scope["splunk"]["idxc"]["replication_port"] = getValueFromDEFAULTS("splunk.idxc.replication_port", casting=int) or 4001
-    vars_scope["splunk"]["enable_service"] = os.environ.get('SPLUNK_ENABLE_SERVICE') or False
-    vars_scope["splunk"]["smartstore"] = getValueFromDEFAULTS("splunk.smartstore")
-
-    vars_scope["splunk"]["app_paths"] = {}
-    if PLATFORM == "linux":
-        vars_scope["splunk"]["group"] = getValueFromDEFAULTS("splunk.group") or "splunk"
-        vars_scope["splunk"]["exec"] = getValueFromDEFAULTS(
```

Comment:
The main reasons for these being defined in here is to have python handle the pathing for different OS's.  My only real concern is windows support.